File: Factorial.java
import java.util.*; 
import java.io.*;

class Function {  
 	int FirstFactorial(int num) { 
		if (num != 1) {
			return num*FirstFactorial(num - 1);
		} 
		return num;		
	}  
  
  public static void main (String[] args) {  
    // keep this function call here     
    Scanner  s = new Scanner(System.in);
    Function c = new Function();
    System.out.print(c.FirstFactorial(s.nextLine())); 
  }   
  
}

File: Factorial.php
<?php 

function FirstFactorial($num) {  

    if ($num >= 1 && $num <= 18) {
      
        $total = $num;  
    
        for ($i = 1; $i < $num; $i++) {
            $total = $total * $i;
        }
        
        return $total;
    }
    
    return false;
         
}